### ISSUE TITLE
NavItem does not accept many children anymore

### ISSUE THREAD

**Issue by gxapplications** [react-materialize/react-materialize#334]
1.0.9 has a breaking change: The following code is not possible anymore :

```
<NavItem href='#settings-modal'>
  <Icon>settings</Icon>
  <span className='hide-on-large-only'>Settings</span>
</NavItem>
```

This code was my way to have a full icon+text item in mobile size (side nav mode) and only icon in desktop size.
If I put a div or a span to wrap NavItem content, then styles are broken (no margin, no pointer style)...

**Comment by alextrastero** [comment]
Hey @gxapplications have a look at the linked PR please

**Comment by gxapplications** [comment]
Good solution for me :)

**Comment by alextrastero** [comment]
Published, try 1.0.10

**Comment by lloydwatkin** [comment]
I've updated my syntax to match the above which fixes things but I originally had:

```jsx
<NavItem onClick={this._gotoSettings}>
  <Icon>settings</Icon> Settings
</NavItem>
```

Which caused:

![screenshot from 2017-08-22 12-12-10](https://user-images.githubusercontent.com/271622/29567082-099c2b7a-8744-11e7-8df7-24b0e95046da.png)
> Element type is invalid: expected a string

Workaround is to add a `<span/>` as above.

Adding comment not in order to get a fix, but in case anyone hits the same thing.
